charts/assets/templates/deployment.yaml

```diff
@@ -0,0 +1,66 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "assets.fullname" . }}
+  labels:
+    {{- include "assets.labels" . | nindent 4 }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      {{- include "assets.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+    {{- with .Values.podAnnotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+    {{- end }}
+      labels:
+        {{- include "assets.selectorLabels" . | nindent 8 }}
+    spec:
+      serviceAccountName: {{ include "assets.serviceAccountName" . }}
```

Comment:
If we are not accessing the kubernetes API better to not use any service account.
```suggestion
```